app/controllers/admin/cache_settings_controller.rb

```diff
@@ -3,7 +3,7 @@
 module Admin
   class CacheSettingsController < Spree::Admin::BaseController
     def edit
-      @results = Exchange.cachable.map do |exchange|
+      @results = Exchange.cachable.limit(5).map do |exchange|
```

Comment:
what if the 6th exchange is the one with important diffs?

Reply:
I'm wondering the same. I think the really slow thing on this page is running the `ProductsCacheIntegrityChecker` on every order cycle. To generate the diff it compiles the json from scratch. The best idea I have would be to just list the cached order cycles and link them to another view that shows only that one. But that's a new feature.